pinot-common/src/main/java/org/apache/pinot/common/function/AggregationFunctionType.java

```diff
@@ -101,6 +101,13 @@ public static AggregationFunctionType getAggregationFunctionType(String function
       try {
         return AggregationFunctionType.valueOf(upperCaseFunctionName);
       } catch (Exception e) {
+        if (upperCaseFunctionName.contains("_")) {
```

Comment:
This won't cover functions such as `percentile_tdigest`.
We can make the current one a private helper method, and add a new `getAggregationFunctionType` to look up for both functionName with/without underscore